blockchain/validators/src/metadata/avr/data/sample_ias_records.toml

```diff
@@ -0,0 +1,99 @@
+[[node]]
+responder_id = 'mc://node1.prod.mobilecoinww.com/'
+first_block_index = 0
+last_block_index = 1
+
+[[node]]
+responder_id = 'mc://node1.prod.mobilecoinww.com/'
+first_block_index = 1
```

Comment:
> @eranrund commented
> Doesn't this mean it overlaps with the previous range?

Reply:
Yeah, though this seems to happen only on the origin block. I have to smooth out this issue yet.